easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb

```diff
@@ -0,0 +1,83 @@
+easyblock = 'PythonPackage'
+
+name = 'SCGid'
+version = '0.9b0'
+# No tagged commit for SCGid 0.9b, taking commit from commit history
+local_commit = 'be106e4cd888fd913a527ea821e510e0afb100be'
+
+homepage = 'https://github.com/amsesk/SCGid'
+description = 'A consensus approach to contig filtering and genome prediction from single-cell sequencing libraries'
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ["https://github.com/amsesk/SCGid/archive"]
+sources = ['%s.tar.gz' % local_commit]
+patches = [
+    "SCGid-0.9-fixPrintToPython3.patch",
+    "SCGid-0.9b0-initEdit.patch"
+]
+checksums = [
+    # be106e4cd888fd913a527ea821e510e0afb100be.tar.gz
+    '1f625e8f87b4d5e4ec9831ebe48763676679d6d6c41b05ef0bb59e6a7f839153',
+    'ee251b71ed3eed097b05751a3cea302d954000e1a8112e6e3f297e67c84996f7',  # SCGid-0.9-fixPrintToPython3.patch
+    '5deb9911beef69bc127b966f94199511800069737f822e15ca5115a16d8b50c5',  # SCGid-0.9b0-initEdit.patch
+]
+
+local_clams_commit = "e326c34307803b9ea8ffc975ddd9bd05643af931"
+
+builddependencies = [('Python', '3.9.6')]
+
+dependencies = [
+    ('R', '4.1.2'),
+    ('BLAST+', '2.12.0'),
+    ('AUGUSTUS', '3.4.0'),
+    ('Java', '11', '', True),
+    ('Maven', '3.6.3', '', True),
+    ('plotly.py', '5.4.0'),
+    ('ETE', '3.1.2'),
+    ('PyYAML', '5.4.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = False
+
+exts_defaultclass = "Tarball"
+local_config_path = "%(installdir)s/lib/python3.9/site-packages/config.yaml"
```

Comment:
please use `/python%(pyshortver)s/`

or just remove, `local_config_path` is not used anymore